docs/01-getting-started-script.md

```diff
@@ -0,0 +1,44 @@
+# Getting Started Script
+
+The provided account setup script creates and configures all of the required Azure resources.
+
+The script will create and configure the following resources:
+- Resource group
+- Storage account
```

Comment:
Should we add a vnet?
```sh
Virtual Network (optional)
```

Reply:
Possibly, but I'm not sure how we'd do output. We could potentially just leave this out until we have "environments"